doc/source/whatsnew/v0.21.1.txt

```diff
@@ -147,4 +147,4 @@ Other
 ^^^^^
 
 -
```

Comment:
can you revert this file

Reply:
HI Jeff, I'm afraid my lack of git experience is showing as this file has been a bit of a thorn in my side.   I'm concerned that my attempts to fix my mess will just result in wasting more of your time.  Would you mind sharing the syntax necessary to revert (or reset?) this file to the proper commit?

Reply:
sure ill fix this upl